src/App.js

```diff
@@ -7,7 +7,8 @@ import CreepypastaPage from './pages/CreepypastaPage';
 import FullCreepypastaPage from './pages/FullCreepypastaPage';
 import CreatePage from './pages/CreatePage';
 import ProfilePage from './pages/ProfilePage';
-import ProfilePageBookmark from './pages/ProfileBookmarkPage';
+import ProfilePageBookmarkPage from './pages/ProfileBookmarkPage';
```

Comment:
You are using 3 different names for the same component, it would be best if you choose one name and use it everywhere (as the js file name, the component name and as the import name)